docs/plugins/User-certificates-plugin.md

```diff
@@ -0,0 +1,225 @@
+# User Certificates Plugin 
+The User Certificates plugin enhances security by allowing authentication through both a username/password and an x509 user certificate. User certificates work across any cluster with a shared root Certificate Authority (CA) with the user's certificate. This means that you can have a single user certificate that is valid across multiple clusters.
+
+::: tip
+The User Certificates plugin is included in commercial versions of EventStoreDB.
+:::
+
+
+# Setup Instructions
+
+1. Download the User Certificates plugin.
+2. Place it in the `.\plugins` directory within your EventStoreDB installation. 
+3. In the EventStoreDB `.\config` directory, add the following configuration to `.\config\plugin-config.json`:
+
+```
+{
+	"EventStore": {
+		"Plugins": {
+			"UserCertificates": {
+				"Enabled": true
+			}
+		}
+	}
+}
+```
+
+After installation and configuration, the server logs an entry confirming the plugin's activation:
+
+```
+[13260, 8,13:59:10.445,INF] UserCertificatesPlugin: user X.509 certificate authentication is enabled
+```
+
+## Connecting with a User Certificate
+
+Use the following command as an example to connect using `curl`: 
+
+```
+curl -i https://localhost:2113/streams/%24all --cert user-admin.crt --key user-admin.key
+```
+
+## Creating a User Certificate
+
+The user certificate must adhere to the following requirements:
+
+- The certificate has a root CA in common with the node certificate.
+- The root CA that they have in common is trusted by the node.
+- The CN of the certificate is the same as the username.
+- The certificate has the ClientAuth EKU, and not the ServerAuth EKU.
+- The certificate must be in date.
+- The CN is the username of a user that exists in the database.
+
+> Note: Certificates can also authenticate `admin` and `ops` users.
```

Comment:
this `>` will just make a blockquote. I suggest turning this into a `::: tip` box

Reply:
Good eye @rsowald